app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,32 @@
+class OrdersController < ApplicationController
+  def show
+    @order = Order.find_by(id: session[:order_id])
+    render :show
+  end
+
+  def edit
+    @order = Order.find_by(id: session[:order_id])
+    @user = User.find_by(id: session[:user_id])
+    render :edit
+  end
+
+  def update
+    @order = Order.find_by(id: session[:order_id])
+    @order.assign_attributes(order_update_params[:order])
+    @order.assign_attributes(order_state: "paid")
```

Comment:
If I get a chance, I'll pull down your code and poke at it to make sure I've got the syntax right and then comment with recommendations once I'm 100% sure I get it.